site-packages/integralstor_utils/django_utils.py

```diff
@@ -0,0 +1,43 @@
+
+def in_request_parameters(request, param_name):
```

Comment:
Since `param_name` is probably passed as string, and not a list, shouldn't `in_request_parameters` be named `in_request_parameter` instead? 

This function asserts/checks the presence of a parameter in a request, so, may be it's better if it's named something of sorts like `is_in_request_parameter` or `assert_request_parameter`?